### PR TITLE
Submit Media Grid snippet for approval

### PR DESCRIPTION
**Pull request by zremboldt**
I've addressed all PR feedback. Let me know what you think.
Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -375,6 +376,19 @@ Give your images a shadow and rounded corners.
...
+## [Media Grid](https://github.com/zremboldt/obsidian-media-grid)
+
+by [@zremboldt](https://github.com/zremboldt)
+
+Add more meaning and visual interest to your documents by easily grouping sets of images, videos, or even audio tracks into grid layouts.  
+I've linked to my repo for a more thorough explanation of custom settings, requirements, how it works, etc.
+
+![](media/css-snippets/media-grid.png)
+
+[Obsidian Media Grid repo](https://github.com/zremboldt/obsidian-media-grid)
```
Comment (kmaasrud) [inline README.md:388]: Follow the conventions of `[📁 filename.css](<link to snippet>)` and add [this file](https://github.com/zremboldt/obsidian-media-grid/blob/main/media-grid-snippet.css) inside `code/css-snippets`.

File: README.md [hunk 2]
```diff
@@ -375,6 +376,19 @@ Give your images a shadow and rounded corners.
 
 ---
 
+## [Media Grid](https://github.com/zremboldt/obsidian-media-grid)
+
+by [@zremboldt](https://github.com/zremboldt)
+
+Add more meaning and visual interest to your documents by easily grouping sets of images, videos, or even audio tracks into grid layouts.  
+I've linked to my repo for a more thorough explanation of custom settings, requirements, how it works, etc.
```
Comment (kmaasrud) [inline README.md:384]: You are welcome to write a longer explanation if you deem it necessary. In the end you can add something like:

```markdown
For more information on how to use this snippet, [see it's original repo](https://github.com/zremboldt/obsidian-media-grid).
```

or something similar.

**Review by kmaasrud — CHANGES_REQUESTED** [review]
I would prefer if you follow the conventions of the other CSS snippets and add the snippet as a CSS file under `code/css-snippets`. See my review comments for more info.

File: README.md (hunk 2 above)
Comment (zremboldt) [reply README.md:384]: Sounds good. I'll try to get back to this tonight. Thank you!

**Comment by kmaasrud** [comment]
Awesome work! Thanks for contributing :smile:

**Pull request merged**